src/Owin.Security.Providers.FreeConferenceCall/FreeConferenceCallAuthenticationOptions.cs

```diff
@@ -0,0 +1,145 @@
+
+        /// <summary>
+        ///     Gets or sets timeout value in milliseconds for back channel communications with FreeConferenceCall.
+        /// </summary>
+        /// <value>
+        ///     The back channel timeout in milliseconds.
+        /// </value>
+        public TimeSpan BackchannelTimeout { get; set; }
+
+        /// <summary>
+        ///     The request path within the application's base path where the user-agent will be returned.
+        ///     The middleware will process this request when it arrives.
+        ///     Default value is "/signin-freeconferencecall".
+        /// </summary>
+        public PathString CallbackPath { get; set; }
+
+        /// <summary>
+        ///     Get or sets the text that the user can display on a sign in user interface.
+        /// </summary>
+        public string Caption
+        {
+            get { return Description.Caption; }
+            set { Description.Caption = value; }
+        }
+
+        /// <summary>
+        ///     Gets or sets the FreeConferenceCall supplied Client ID
+        /// </summary>
+        public string ClientId { get; set; }
+
+        /// <summary>
+        ///     Gets or sets the FreeConferenceCall supplied Client Secret
+        /// </summary>
+        public string ClientSecret { get; set; }
+
+        /// <summary>
+        /// Gets the sets of OAuth endpoints used to authenticate against FreeConferenceCall.  Overriding these endpoints allows you to use FreeConferenceCall Enterprise for
+        /// authentication.
+        /// </summary>
+        public FreeConferenceCallAuthenticationEndpoints Endpoints { get; set; }
+
+        /// <summary>
+        ///     Gets or sets the <see cref="IFreeConferenceCallAuthenticationProvider" /> used in the authentication events
+        /// </summary>
+        public IFreeConferenceCallAuthenticationProvider Provider { get; set; }
+
+        /// <summary>
+        /// A list of permissions to request.
+        /// </summary>
+        public IList<string> Scope { get; private set; }
+
+        /// <summary>
+        ///     Gets or sets the name of another authentication middleware which will be responsible for actually issuing a user
+        ///     <see cref="System.Security.Claims.ClaimsIdentity" />.
+        /// </summary>
+        public string SignInAsAuthenticationType { get; set; }
+
+        /// <summary>
+        ///     Gets or sets the type used to secure data handled by the middleware.
+        /// </summary>
+        public ISecureDataFormat<AuthenticationProperties> StateDataFormat { get; set; }
+
+        /// <summary>
+        ///     Initializes a new <see cref="FreeConferenceCallAuthenticationOptions" />
+        /// </summary>
+        public FreeConferenceCallAuthenticationOptions()
+            : base("FreeConferenceCall")
+        {
+            Caption = Constants.DefaultAuthenticationType;
+            CallbackPath = new PathString("/signin-freeconferencecall");
```

Comment:
Not sure if this CallbackPath specified here can be anything if it if has restrictions.